src/routes.ts

```diff
@@ -44,6 +45,12 @@ const Routes: Route<any>[] = [
     component: PlayoffBracket,
     usedIn: ['playoff'],
   },
+  {
+    name: 'Rankings Ticker (Audience Display)',
+    url: '/ranking/ticker',
+    component: KeyableTicker,
+    usedIn: ['playoff'],
```

Comment:
This usedIn determines whether it shows up in the automated setup dropdown. I think we'd actually want an empty array here but maybe run locally and double check?
```suggestion
    usedIn: [],
```

Reply:
Resolved in [f425ecf](https://github.com/evanlihou/fim-queueing/pull/41/commits/f425ecfc36195d9c01a374c99d50b75e5fdaaa6d)